ingest_graph_validator/actions/ingest_validator_action.py

```diff
@@ -78,20 +79,17 @@ def __attempt_validation(self, submission, sub_uuid):
             if validation_result is not None:   
                 self._logger.info(f"validation finished for {sub_uuid}")
 
-                if validation_result["valid"]:
-                    self._ingest_api.put(f'{submission_url}/graphValidEvent', data=None)
-                else:
-                    self._ingest_api.put(f'{submission_url}/graphInvalidEvent', data=None)
-
+                if not validation_result["valid"]:
                     for failure in validation_result["failures"]:
                         for entity in failure['affectedEntities']:
                             self.__patch_entity(failure['message'], entity['link'])
-
+                    self._ingest_api.put(f'{submission_url}/graphInvalidEvent', data=None)
+                else:
+                    self._ingest_api.put(f'{submission_url}/graphValidEvent', data=None)
                 self._logger.info(f'Finished validating {sub_uuid}.')
         except Exception as e:
             self._logger.error(f"Failed validation with error {e}.")
-            self._logger.info("Reverting submission graphValidationState to Pending")
-            self._ingest_api.put(f'{submission_url}/graphPendingEvent', data=None)
+            # TODO add endpoint to restore submission to metadata valid and log error
```

Comment:
is this todo still needed?

Reply:
Yes, I'm gonna make a ticket for this

Reply:
https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/568